ledger/participant-state/kvutils/src/test/suite/scala/com/daml/ledger/participant/state/kvutils/committer/transaction/TransactionCommitterSpec.scala

```diff
@@ -326,11 +355,14 @@ class TransactionCommitterSpec
       val cid = builder.newCid
 
       val (expectedContractInstance, txEntry) = txEntryWithDivulgedContract(builder, cid)
-
+      val txEntryBuilder = txEntry.toBuilder
```

Comment:
It would be ideal if we didn't need this change.

Reply:
The problem here is that deduplication duration should always be set. Do we want to remove that restriction and add a fallback?

Reply:
Will add the comment. 
The `blind` though implicitly set the `dedupValue` so setting it sort of makes sense.